tests/explainers/test_tree.py

```diff
@@ -1714,3 +1714,39 @@ def test_lightgbm_interactions():
 
     explanation = explainer(X, interactions=True)
     assert np.allclose(np.stack(explanation.values, axis=-1).sum(axis=(1, 2)) + explanation.base_values, predicted)
+
+
+def test_catboost_column_names_with_special_characters():
+    # GH #3475
+    # Seed
+    catboost = pytest.importorskip("catboost")
+    np.random.seed(42)
+
+    # Simulate a dataset
+    data = {
+            'x1': np.random.rand(1000),
+            'x2': np.random.choice([0, 1], size=1000),
+            'x3': np.random.choice([0, 1], size=1000),
+            'x4': np.random.choice([0, 1], size=1000),
+            'x5=ROMÁNIA': np.random.choice([0, 1], size=1000),
+            'label': np.random.choice([0, 1], size=1000)
+        }
```

Comment:
Nitpick: I think the unit test can be simplified somewhat to be a minimal reproducer of the error. For example, we drop x2, x3 & x3; and perhaps use just use say 10 rather than 1000 rows?

Reply:
fixed. If you could kindly merge the changes.